website/blog/2025-06-16-the-new-dbt-vscode-extension.md

```diff
@@ -0,0 +1,110 @@
+
+After using it, it’s hard to imagine going back. Working with dbt in VS Code without this extension just doesn’t make sense anymore.
+
+It comes with a lot of features to streamline your work and make you more efficient by developing faster and spending less. But let me tell you about my favorites:
+
+### Catch SQL Errors in Real Time
+
+There was no question what I was picking first. No more waiting for your platform to debug your code for you. If you misspell a column name or goof up a function's order of parameters, you catch those errors before you run anything.
+
+This is because Fusion doesn't treat SQL code as just a string anymore; it really understands it. It also shows you some helpful information about the error.
+
+<Lightbox src="/img/blog/2025-06-06-the-new-dbt-vscode-extension/vs_code_extension_function_error.png" title="Showing function errors." />
+
+<Lightbox src="/img/blog/2025-06-06-the-new-dbt-vscode-extension/vs_code_extension_column_error.png" title="Showing column name errors." />
+
+This is the greatest improvement of this engine, IMHO.
+
+### Model and Column Lineage
+
+My next favorite feature is the lineage view. If you were a <Constant name="cloud" /> platform user, you would feel at home. And if you were using dbt Core, finally, no more generating `dbt docs` to visualize lineage.
+
+Now there's a tab lineage tab that shows your project’s lineage directly in VS Code. It’s interactive and live. You can use the lenses feature, that's pretty cool to have a good visualization of your project by different attributes like resource_type, or materialization.
+
+<Lightbox src="/img/blog/2025-06-06-the-new-dbt-vscode-extension/vs_code_extension_project_lineage.png" title="Project Lineage." />
+
+And something I was not expecting to be here, but thankfully it is, column-level lineage! Not just where columns come from, but also how they change: renamed, transformed, or passed through.
+
+This is incredibly helpful for debugging transformations or understanding how that key metric is shaped across models.
+
+<Lightbox src="/img/blog/2025-06-06-the-new-dbt-vscode-extension/vs_code_extension_cll.png" title="Column-level Lineage." />
+
+### Instant refactoring
+
+Ok, let me show you just one more thing! Have you ever faced a situation where you'd like to rename a model or a column, but it's used many places downstream that you give up because you don't want to refactor everything or you are afraid you will break something?
+
+Now, thanks to the deep dbt Fusion SQL understanding, you can rename your model or column, and the extension will refactor all downstream dependencies for you. But don't worry, before doing it, the extension allows you to see a preview of the changes, so you can be sure it is doing what you want.
+
+<video width="100%" height="100%" playsinline muted controls>
+  <source src="/img/blog/2025-06-06-the-new-dbt-vscode-extension/vs_code_extension_refactoring.webm" type="video/webm" />
+</video>
```

Comment:
@bruno-szdl did you ever see this working?

[the preview shows](https://docs-getdbt-1pxx8hflz-dbt-labs.vercel.app/blog/vscode-extension-experience) that the video isn't rendering in Vercel

did this video ever work for you by chance?

<img width="907" alt="image" src="https://github.com/user-attachments/assets/ded300db-0b00-43a0-bfef-4dd0105098c8" />